libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -244,10 +254,15 @@ bool AP_RangeFinder_LightWareI2C::sf20_init()
     // version strings for reporting
     char version[15] {};
 
-    sf20_get_version("?P\r\n", "p:", version);
-
-    if (version[0]) {
-        hal.console->printf("SF20 Lidar version %s\n", version);
+    // Attempt to get lidar version up to MAX_VERSION_STRING_RETRY_ATTEMPTS times. This allows the UART to flush in between inits.
+    memset(version, 0, sizeof(version));
```

Comment:
Instead of the memset here, could you set the array to 0 through aggregate initialization above?
`char version[15] {0};`

Reply:
Done!